assets/css/_geolocation.scss

```diff
@@ -33,7 +26,26 @@
   }
 
   .leaflet-container {
-    border: 4.5px solid #000;
     height: 500px;
   }
+
+  .geolocation-button {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
```

Comment:
Let's use the `@include appearance(none);` mixin here. See here: https://github.com/streetmix/streetmix/blob/master/assets/css/_mixins.scss#L9